flepimop/gempyor_pkg/src/gempyor/outcomes.py

```diff
@@ -91,6 +91,8 @@ def onerun_delayframe_outcomes(
     load_ID: bool = False,
     sim_id2load: int = None,
 ):
+    np.random.seed(seed=sim_id2write)
```

Comment:
This might fix a seed in a way that is not always desirable: with classical inference, every ith iteration will have the same seed across slots. I would rely on /dev/random for now to be sure things are working correctly and keep the reproducibility of runs as a follow-up issue.

Reply:
that suggests a fix is needed in how classical inference works - any inference method should be managing set seeding in a sensible way.

Reply:
I fully agree. I would suggest a plan as follows:
1. Make thing works correctly as people expect them (so doing the real random thing that the emcee is doing also here)
2. Streamline the pathway to call gempyor (https://github.com/HopkinsIDD/flepiMoP/discussions/463#discussioncomment-11821663 shows 3 ways gempyor is called, but we should have one)
3. Build something with seed so that the run with the above pathway is reproducible.

If we do 3. before 2. there might be places that have unexpected behavior which is IMO more critical than being reproducible (even though that's also very important)

Reply:
Addressed in 48c1bf4c12486a8024e4804b5ba4ff0b4b7e65ab and c31aa840703fa932c7a69e026f04a8706833c277.